hyperspy/tests/model/test_edsmodel.py

```diff
@@ -250,7 +250,8 @@ def setup_method(self, method):
     def test_lines_intensity(self):
         s = self.s
         m = s.create_model()
-        m.multifit()    # m.fit() is just too inaccurate
+        # HyperSpy 2.0: remove setting iterpath='serpentine'
+        m.multifit(iterpath='serpentine')    # m.fit() is just too inaccurate
```

Comment:
I know it's an old comment, but can we quantify what we mean by `# m.fit() is just too inaccurate`

Reply:
Indeed, this is a good point, that sounds odd to me and it would be good to do something about it!

Reply:
It look to me that this statement is not correct now and we can simply removed it. I added a test to confirm this but I think we could also remove this test comparing `fit` and `multifit`.
This comment was commited 4 years ago (https://github.com/hyperspy/hyperspy/commit/5b3b92f3ddf4788a08df86129dee1c7cdb881be8) and the commit message doesn't say what could the reason for this comment, so I don't see any reason to keep it, unless it is possible to demonstrate that it doesn't work.

Reply:
If the test is cheap to run, I'd vote to keep it.